### PR TITLE
Fix info button arrangement in the data table component

### LINKED ISSUE #2283: Adjust info button in the data table component
**Is your feature request related to a problem? Please describe.**
The optional info button which can be set in the table header cell is currently 24px big and is placed left to the label. This causes a bad looking alignment with the rest of the table cells and due to the size, the padding of the header ell seems to be ignored

**Describe the solution you'd like**
- Set the size of the info button to **16px**
- Place the info button on the **right side** of the label
- The margin between the label and the info button should be **8px.**
image

### PR DESCRIPTION
**Pull request by ddouglasz**
This PR fixes the issue opened regarding data-table info button arrangement. 
image

Fixes #2283

### CODE REVIEW

File: .changeset/slimy-walls-worry.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/data-table': minor
```
Comment (emmenko) [inline .changeset/slimy-walls-worry.md:2]: Please be careful when to use which version types. Minor is mostly used for new features or some substantial changes. For bug fixes, minor changes, normal dep updates, etc. we should use `patch`.

```suggestion
'@commercetools-uikit/data-table': patch
```

File: .changeset/slimy-walls-worry.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+---
+
+Fix arrangement of data-table info button.
+
+The optional info button which can be set in the table header cell is placed left to the label. This causes a bad looking alignment with the rest of the table cells and due to the size, the padding of the header cell seems to be ignored.
```
Comment (emmenko) [inline .changeset/slimy-walls-worry.md:7]:
```suggestion
Fix position of header cell icon to the right of the text.
```

**Review by emmenko — COMMENTED** [review]
Thanks for the fix! I left a couple of comments about the changeset otherwise 👌

File: .changeset/slimy-walls-worry.md (hunk 1 above)
Comment (ddouglasz) [reply .changeset/slimy-walls-worry.md]: > FYI: if a PR closes or fixes an issue, we should use the syntax `Closes` or `Fixes`, which then instructs GitHub to automatically close the issue.
> 
> image

Thank you. I get it now. Updated here:

https://github.com/commercetools/ui-kit/pull/2284/commits/13f01a6db493060ec99c82a2e33e8f846aa07727

**Review by emmenko — APPROVED** [review]
Thanks! 👌

**Review by Rhotimee — APPROVED** [review]

**Review by kark — APPROVED** [review]

**Pull request merged**